internal/cri/annotations/annotations.go

```diff
@@ -106,19 +106,22 @@ func DefaultCRIAnnotations(
 		customopts.WithAnnotation(SandboxNamespace, config.GetMetadata().GetNamespace()),
 		customopts.WithAnnotation(SandboxUID, config.GetMetadata().GetUid()),
 		customopts.WithAnnotation(SandboxName, config.GetMetadata().GetName()),
+		customopts.WithAnnotation(ImageName, imageName),
 	}
 	ctrType := ContainerTypeContainer
 	if sandbox {
 		ctrType = ContainerTypeSandbox
 		// Sandbox log dir only gets passed for sandboxes, the other metadata always
 		// gets sent however.
-		opts = append(opts, customopts.WithAnnotation(SandboxLogDir, config.GetLogDirectory()))
+		opts = append(
+			opts,
+			customopts.WithAnnotation(SandboxLogDir, config.GetLogDirectory()),
```

Comment:
should be adding 
Let's create a new key above after line 74:
```
// SandboxPauseImageName is the name of the contain image used to scope pod shared resources used by the pod's containers, if nil a pause container is not being used.
SandboxImageName = "io.kubernetes.cri.podsandbox.image-name"
```

Then add the new annotation here:
customopts.WithAnnotation(SandboxImageName,imageName),

Reply:
Thanks for your advice.  I have made the change.

Reply:
sorry about my typo on the comment string :)